underscore.js

```diff
@@ -1279,9 +1288,7 @@
 
   // Is a given value an array?
   // Delegates to ECMA5's native Array.isArray
-  _.isArray = nativeIsArray || function(obj) {
-    return toString.call(obj) === '[object Array]';
-  };
+  _.isArray = isArray;
```

Comment:
:/

What if we re-arranged the sections so that the `_.property` parts comes after `_.is[...]`?